src/pages/Settings/AccountsSettingsSection.tsx

```diff
@@ -0,0 +1,157 @@
+// Copyright 2018 - 2021 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
+import { useContext, useState } from 'react'
+import { getStorage } from 'alephium-js'
+
+import { GlobalContext } from '../../App'
+import { Button } from '../../components/Buttons'
+import { BoxContainer, Section } from '../../components/PageComponents/PageContainers'
+import AccountRemovalModal from './AccountRemovalModal'
+import SecretPhraseModal from './SecretPhraseModal'
+import InfoBox from '../../components/InfoBox'
+import styled from 'styled-components'
+import { Trash } from 'lucide-react'
+import { HorizontalDivider } from '../../components/PageComponents/HorizontalDivider'
+
+const Storage = getStorage()
+
+const AccountsSettingsSection = () => {
+  const { currentUsername, wallet, setWallet } = useContext(GlobalContext)
+  const [isDisplayingSecretModal, setIsDisplayingSecretModal] = useState(false)
+  const [isShowingRemoveModalForAccount, setIsShowingRemoveModalForAccount] = useState<string | false>(false)
```

Comment:
This kind of breaks the mental model that every variable that begins with `is...` is a `boolean` (and not a username). Took me a bit to understand what is going on later in this file. Would u consider refactoring? Not a big issue, just an observation.